experimental/framework/fluid-framework/README.md

```diff
@@ -1,4 +1,4 @@
-# @fluid-experimental/experimental-fluidframework
+# fluid-framework
```

Comment:
If we're sticking with @fluid-experimental scope it's probably not necessary right this moment, but I do still think the rest of the package name can be misleading/confusing and it would be helpful to flesh out the readme a bit more to describe how this "fluid-framework" is different from the rest of fluid-framework